src/seeder.js

```diff
@@ -1,27 +1,30 @@
-import config from 'config'
-import Hyperswarm from 'hyperswarm'
-import Corestore from 'corestore'
-import Hyperbee from 'hyperbee'
-import ms from './time-to-milliseconds.js'
-import logger from './logger.js'
-
-export default class Seeder {
+const config = require('config')
+const Hyperswarm = require('hyperswarm')
+const Corestore = require('corestore')
+const Hyperbee = require('hyperbee')
+const ms = require('./time-to-milliseconds.js')
+const logger = require('./logger.js')
+const RAM = require('random-access-memory')
+
+const DEFAULT_DB_NAME = "Hyperbee DB"
+const DEFAULT_SEEDERS_TOPIC = Buffer.from('3b9f8ccd062ca9fc0b7dd407b4cd287ca6e2d8b32f046d7958fa7bea4d78fd75', 'hex')
+
+class Seeder {
     /**
      * @param {object} [opts]
      * @param {Array<{host: string, port: number}>} [opts.bootstrap]
-     * @param {any} [opts.storage]
+     * @param {string} [opts.storage] corestore storage directory 
+     * @param {Uint8Array} [opts.seed] seed for generating the Hyperswarm DHT keyPair
+     * @param {Uint8Array} [opts.topic] seed for generating the Hyperswarm DHT keyPair
+     * @param {string} [opts.dbName] name of the Hyperbee DB
      */
-    constructor(opts) {
+    constructor(opts = {}) {
         // Setup Corestore
-        this.store = new Corestore(opts?.storage || config.get('store.path'), { _autoReplicate: false })
+        this.store = new Corestore(opts.storage || RAM, { _autoReplicate: false })
 
         // Setup Hyperswarm
-        // Use a seed (if configured) to generate the same Hyperswarm DHT keyPair 
-        let seed;
-        try {
-            seed = Buffer.from(config.get('hyperswarm.seed'), 'hex')
-        } catch {}
-        this.swarm = new Hyperswarm({...opts, seed})
+        this.swarm = new Hyperswarm(opts)
```

Comment:
I think will be better to have `opts.hyperSwarmOptions` or something like this (probably `opts.swarm` for consistency with above